scripts/delete.py

```diff
@@ -1,6 +1,6 @@
 import logging
 
-from scripts.util import documents
+from scripts.util import documents_v2 as documents
```

Comment:
Correct me if I'm wrong but `documents` is a generator and `documents_v2` is a migration script?
It doesn't look like they should be drop in replacements of each other